paddle/fluid/inference/utils/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
 cc_library(benchmark SRCS benchmark.cc DEPS enforce)
 cc_test(test_benchmark SRCS benchmark_tester.cc DEPS benchmark)
-cc_binary(visualizer SRCS visualizer.cc DEPS analysis
-    paddle_pass_builder ir_pass_manager pass graph_viz_pass analysis_passes)
+#cc_binary(visualizer SRCS visualizer.cc DEPS analysis
```

Comment:
why change this?

Reply:
sorry, this change will be fixed in [](https://github.com/PaddlePaddle/Paddle/pull/15575)